docs/machine-learning/resources/tasks.md

```diff
@@ -230,3 +229,27 @@ The forecasting task use past time-series data to make predictions about future
 You can train a forecasting model with the following algorithm:
 
 <xref:Microsoft.ML.TimeSeriesCatalog.ForecastBySsa%2A>
+
+## Image Classification
+
+A [supervised machine learning](glossary.md#supervised-machine-learning) task that is used to predict the class (category) of an image. The input is a set of labeled examples. Each label normally starts as text. It is then run through the TermTransform, which converts it to the Key (numeric) type. The output of the image classification algorithm is a classifier, which you can use to predict the class of new images. Examples of image classification scenarios include:
+
+* Determining the breed of a dog as a "Siberian Husky", "Golden Retriever", "Poodle", etc.
+* Determining if a manufacturing product is defective or not.
+* Determining what types of flowers as "Rose", "Sunflower", etc.
+
+### Image classification trainers
+
+You can train an image classification model using the following training algorithms:
+
+* <xref:Microsoft.ML.Vision.ImageClassificationTrainer>
+
+## Object Detection
+
+A [supervised machine learning](glossary.md#supervised-machine-learning) task that is used to predict the class (category) of an image but also gives a bounding box to where that category is within the image. Examples of object detection include:
```

Comment:
Consider clarifying that as opposed to image classification, you're classifying multiple objects in an image instead of just one. https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/object-detection-onnx#what-is-object-detection